src/backend/drakula/database.py

```diff
@@ -1,74 +1,96 @@
+import uuid
+from dotenv import load_dotenv
 from os import getenv
+import os
 from typing import Optional
 from hashlib import md5
-
-from mysql.connector import connect
-
+from mysql.connector import connect, Error
 from .models import Airport
 
 DEFAULT_AIRPORT_AMOUT = 15
 
+load_dotenv()
```

Comment:
The environment for the database is loaded in multiple places now, here and in the main. Why is the environment loaded here?

Reply:
done